projects/core/components/textfield/textfield.component.ts

```diff
@@ -81,6 +84,21 @@ export class TuiTextfieldComponent<T> implements TuiDataListHost<T> {
 
     protected readonly icons = inject(TUI_COMMON_ICONS);
 
+    protected computedFiller = computed(() => {
+        const value = this.directive?.nativeValue() || '';
+        const filledValue = value + this.filler().slice(value.length);
+
+        return filledValue.length > value.length ? filledValue : '';
+    });
+
+    protected showFiller = computed(() =>
+        Boolean(
+            this.focused() &&
+                this.computedFiller() &&
```

Comment:
Weird tabbing 🤔

Reply:
Unfortunately, prettier does it (

https://github.com/user-attachments/assets/a06f4c4c-a307-405d-a285-c05886116f6d

The only way to bypass it is drop `Boolean` and add some `!!`

```ts
protected showFiller = computed(
    () =>
        this.focused() &&
        !!this.computedFiller() &&
        (!!this.directive?.nativeValue() || !this.input?.nativeElement.placeholder),
);
```

What do you think ?

Reply:
I guess it looks better

Reply:
You can report this issue to [prettier](https://github.com/prettier/prettier)

Reply:
> You can report this issue to [prettier](https://github.com/prettier/prettier)

@splincode done
https://github.com/prettier/prettier/issues/16709